lib/utils.js

```diff
@@ -22,24 +38,15 @@ const stringify = (obj) => {
             value = value.toString();
         }
 
-        key = key + ' '.repeat(maxKeyLength - key.length) + ' =';
-
-        lines.push(key + ' ' + value);
+        lines.push(key + ' '.repeat(maxKeyLength - key.length) + ' = ' + value);
 
         return lines;
 
     }, []).join('\n');
 };
 
 // Custom Scaled Merger:
-const merge = (objects, is_json) => {
-    is_json = is_json || false;
-    if (is_json) {
-        return objects.reduce((merged, obj) => {
-            merged.push(Object.assign(merged.pop(), obj));
-            return merged;
-        }, [{}])[0];
-    }
+const merge = (objects) => {
```

Comment:
I'm assuming you removed these lines because `is_json` is never set to true anywhere?

Reply:
yeap